modules/virt-template-vmi-probe-config.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-monitoring-vm-health.adoc
+
+[id="virt-template-vmi-probe-config_{context}"]
+= Template: Virtual machine instance configuration file for defining health checks
+
+[source,yaml]
+----
+  apiVersion: kubevirt.io/v1alpha3
```

Comment:
This whole example file is indented by two spaces. I'm not sure if that actually breaks anything, but it is probably best to remove those extra spaces.

Reply:
Removed the extra space